Uplift/Views/GymDetail/GymDetailTimeInfoView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  GymDetailTimeInfoView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/22/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class GymDetailTimeInfoView: UILabel {
+
+    private var facility: Facility
+    private var selectedDayIndex = 0
+    private var displayedText = ""
+    private var timesText = NSMutableAttributedString()
+    private var paragraphStyle = NSMutableParagraphStyle()
+
+    // MARK: - Init
+    init(facility: Facility) {
+        self.facility = facility
+        super.init(frame: CGRect())
+
+        timesText.mutableString.setString(displayedText)
+
+        paragraphStyle.lineSpacing = 5.0
+        paragraphStyle.maximumLineHeight = 26
+        paragraphStyle.alignment = .center
+
+        attributedText = timesText
+        backgroundColor = .primaryWhite
+        numberOfLines = 0
+        font = ._16MontserratRegular
+        textColor = .primaryBlack
+        textAlignment = .center
+
+        updateAppearance()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Update
+    private func updateAppearance() {
```

Comment:
why do we use `updateAppearance()` instead of just calling `updateTimes()` directly, given that `updateAppearance` just calls `updateTimes`?

Reply:
When tags are implemented with new backend, `updateAppearance` will call `updateTimes` and `updateTags`